File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
int n;
do{
    n = get_int ("enter height of pyramid \n") ;
    } 
    while (n<1 || n>8) ;
    for (int x = 1; x<=n; x++)
    {
        for (int y = 1; y<=2*n+2; y++){
            int z=y;
            if (z<=n-x)
                {printf(" ");}
                else if (z>n-x  && z<=n)
                {printf("#");}
                else if (z>n && z<=n+2)
                {printf(" ");}
                else if (z>n+2 && z<=n+2+x)
                {printf("#");}
                else if (z>n+2 && z<=2*n+2)
                {printf(" ");}

        }
   printf("\n");
    }

}

File: mario.py
import string

spacer = 0
height = 0

while height < 1 or height > 8:
    try:
        height = int (input ("Height: \n"))
    except ValueError:
        print ("Please enter a number")



for x in range (1, height + 1, 1):
    for y in range (0 , height, 1):
        if height - y <= x:
            print ("#", end = '')
        else:
            print (" ", end = '')
    print ("")

